ruby/src/api.cc

```diff
@@ -136,11 +136,7 @@ namespace leatherman { namespace ruby {
         } else {
             LOG_INFO("ruby was already loaded.");
         }
-        try {
-            return unique_ptr<api>(new api(move(library)));
-        } catch (lth_lib::missing_import_exception& ex) {
-            throw ex;
-        }
+        return unique_ptr<api>(new api(move(library)));
```

Comment:
Getting a different error now, `non-const lvalue reference to type 'leatherman::ruby::api' cannot bind to a temporary of type 'leatherman::ruby::api'`. Which I think I understand, but am not sure how to fix correctly.